build-logic/build-platform/build.gradle.kts

```diff
@@ -16,7 +16,7 @@ dependencies {
         // Gradle Plugins
         api("com.gradle:gradle-enterprise-gradle-plugin:3.6.2")
         api("com.gradle.enterprise:test-distribution-gradle-plugin:2.1") // Sync with `settings.gradle.kts`
-        api("org.gradle.guides:gradle-guides-plugin:0.18.1-hotfix")
+        api("org.gradle.guides:gradle-guides-plugin:0.18.0")
```

Comment:
Why did you downgrade the guides plugin here?

Reply:
Because `0.18.1-hotfix` was an ugly hard-coded-fix: https://github.com/gradle/guides/commit/076f037beafd7ce4d8d167beb703648fb266f2cf